pom.xml

```diff
@@ -128,45 +97,51 @@
 					<packagingExcludes>pom.xml</packagingExcludes>
 				</configuration>
 			</plugin>
-			<!-- Plugin to run unit tests -->
+			<!-- run integration tests -->
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-surefire-plugin</artifactId>
+				<artifactId>maven-failsafe-plugin</artifactId>
 				<version>2.18.1</version>
 				<executions>
 					<execution>
-						<phase>test</phase>
-						<id>default-test</id>
+						<phase>integration-test</phase>
+						<id>integration-test</id>
+						<goals>
+							<goal>integration-test</goal>
+						</goals>
 						<configuration>
-							<excludes>
-								<exclude>**/it/**</exclude>
-							</excludes>
-							<reportsDirectory>${project.build.directory}/test-reports/unit</reportsDirectory>
+							<systemPropertyVariables>
+								<liberty.test.port>${testServerHttpPort}</liberty.test.port>
+								<war.context>${warContext}</war.context>
+							</systemPropertyVariables>
 						</configuration>
 					</execution>
+					<execution>
+						<id>verify-results</id>
+						<goals>
+							<goal>verify</goal>
+						</goals>
+					</execution>
 				</executions>
+				<configuration>
+					<summaryFile>${project.build.directory}/test-reports/it/failsafe-summary.xml</summaryFile>
+					<reportsDirectory>${project.build.directory}/test-reports/it</reportsDirectory>
+				</configuration>
 			</plugin>
+			<!-- Plugin to run unit tests -->
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-resources-plugin</artifactId>
-				<version>2.7</version>
+				<artifactId>maven-surefire-plugin</artifactId>
```

Comment:
I think the maven war plugin automatically binds surefire but not failsafe.  So when you remove this plugin configuration, the tests that start or end with `TEST`  will still run.  That is another reason it is important to rename the EndpointTest.java to EndpointIT.java and TestApplication.java to ApplicationIT.java (although it doesn't look like the TestApplication does anything.